### ISSUE TITLE
SDK NuGet fails

### ISSUE THREAD

**Issue by WillemMali** [UXDivers/Gorilla-Player-Support#123]
Hi,

When I install the SDK version 0.9.1 or 0.9.1.2 I get the following error:

```
Attempting to gather dependency information for package 'UXDivers.GorillaPlayer.SDK.0.9.1' with respect to project 'Consumer\ConsumerApp.Droid', targeting 'MonoAndroid,Version=v6.0'
Attempting to resolve dependencies for package 'UXDivers.GorillaPlayer.SDK.0.9.1' with DependencyBehavior 'Lowest'
Resolving actions to install package 'UXDivers.GorillaPlayer.SDK.0.9.1'
Resolved actions to install package 'UXDivers.GorillaPlayer.SDK.0.9.1'
Adding package 'UXDivers.GorillaPlayer.Common.0.9.1' to folder 'D:\work\fortytwotech\belval\packages'
Added package 'UXDivers.GorillaPlayer.Common.0.9.1' to folder 'D:\work\fortytwotech\belval\packages'
Added package 'UXDivers.GorillaPlayer.Common.0.9.1' to 'packages.config'
Successfully installed 'UXDivers.GorillaPlayer.Common 0.9.1' to ConsumerApp.Droid
  GET https://api.nuget.org/packages/uxdivers.gorillaplayer.sdk.0.9.1.nupkg
  OK https://api.nuget.org/packages/uxdivers.gorillaplayer.sdk.0.9.1.nupkg 567ms
Installing UXDivers.GorillaPlayer.SDK 0.9.1.
Adding package 'UXDivers.GorillaPlayer.SDK.0.9.1' to folder 'D:\work\fortytwotech\belval\packages'
Added package 'UXDivers.GorillaPlayer.SDK.0.9.1' to folder 'D:\work\fortytwotech\belval\packages'
Added package 'UXDivers.GorillaPlayer.SDK.0.9.1' to 'packages.config'
Successfully installed 'UXDivers.GorillaPlayer.SDK 0.9.1' to ConsumerApp.Droid
========== Finished ==========
Unable to read package from path 'UXDivers.GorillaPlayer.Common.0.9.1\UXDivers.GorillaPlayer.Common.0.9.1.nupkg'.
  File contains corrupted data.
```

So, the package does install but the nupkg is corrupted (according to Visual Studio Community Edition 2015). Anything I can do to get this fixed?

**Comment by escamoteur** [comment]
Funny, I was just about to open an issue with the same topic.
Problem only occurs in Android. My project actually does compile, but I'm not ablte to use the Package Manager after that.

**Comment by WillemMali** [comment]
Yeah, you can remove it by removing the packages from the "packages" folder in your solution and from pacakges.config in your project, NuGet works again after that.

**Comment by escamoteur** [comment]
Same here. I spend the last hour trying different things.

**Comment by escamoteur** [comment]
Did you try to install an older version?

**Comment by WillemMali** [comment]
I just tried 0.9.1 and 0.9.1.2, you?

I don't really have time to wrestle with this stuff now, have to ship in ~20 more work hours.

**Comment by escamoteur** [comment]
same with me. But as it works on iOS I can get started and hope the developers will take care soon

**Comment by LeoHere** [comment]
Hi, package 0.9.1 has issues. I got the same 'unable to read package from path...' from VS/XS on windows (it does not happen from XS Mac). But that issue was corrected in 0.9.1.2 which was the version used in the release.

I will remove 0.9.1 from listing right now. 

Are you 100% you are having this issue with 0.9.1.2? 

As a workaround you might rename it to zip, extract the assemblies and add references to the assemblies directly. Yes, I know a pain in the ...

**Comment by escamoteur** [comment]
Hi, I veridied it's 0.9.1.2 and it's unfortunately also in iOS

What did you mean with rename it to zip?

**Comment by LeoHere** [comment]
That is weird. Is anyone else having issues with the 0.9.1.2 nuget packages?  

If you go to your packages folder, and then go to the Gorilla player SDK you should have the .nuget file (from the error it seems that the package is downloaded, but it has issues reading the file). That file is actually a zip. Rename it, uncompress it and you will get the SDK assemblies. Then you will need to manually reference to the assemblies.

What version of nuget are you using? What version of VS (including update) and XS are you using?

**Comment by escamoteur** [comment]
Just updated nuget today. VS2015 Update 2

**Comment by LeoHere** [comment]
I have just tested adding 0.9.1.2 to a just created solution, using VS2015 Update 2 with Package Manager 3.4.4 and i can't reproduce the issue :(. Can you please check if the zip is not corrupt by renaming it an unzipping it?

**Comment by escamoteur** [comment]
I updated nuget manager this morning to the same version, now everything works.

**Comment by LeoHere** [comment]
Good to know @escamoteur, do you remember which version did you have before the upgrade?

@WillemMali, can you provide the current version of your package manager and confirm if the issue is fixed for you too?

**Comment by WillemMali** [comment]
Hey, the latest version on NuGet works! It does crash after running a few seconds, but that's the next step.
